--- compat.lua	2019-12-09 17:28:52.811852063 +0000
+++ compat.lua	2019-12-09 17:29:24.255956113 +0000
@@ -8,6 +8,7 @@
 local has_pipeworks_mod = minetest.get_modpath("pipeworks")
 local has_beds_mod = minetest.get_modpath("beds")
 local has_ropes_mod = minetest.get_modpath("ropes")
+local has_ham_radio_mod = minetest.get_modpath("ham_radio")
 
 dofile(MP.."/compat/travelnet.lua")
 dofile(MP.."/compat/locator.lua")
@@ -18,6 +19,7 @@
 dofile(MP.."/compat/telemosaic.lua")
 dofile(MP.."/compat/beds.lua")
 dofile(MP.."/compat/ropes.lua")
+dofile(MP.."/compat/ham_radio.lua")
 
 if has_pipeworks_mod then
 	dofile(MP.."/compat/teleporttube.lua")
@@ -37,6 +39,9 @@
 	elseif name == "telemosaic:beacon" or name == "telemosaic:beacon_protected" then
 		jumpdrive.telemosaic_compat(source_pos, target_pos)
 
+	elseif (name == "ham_radio:transmitter" or name == "ham_radio:beacon") and has_ham_radio_mod then
+		jumpdrive.ham_radio_compat(source_pos, target_pos)
+
 	end
 end
 
